.circleci/config.yml

```diff
@@ -108,7 +108,7 @@ jobs:
       - run:
           name: Lint
           command: |
-            $HOME/.local/bin/golangci-lint run -v \
+            $HOME/.local/bin/golangci-lint run -v --skip-dirs-use-default=false\
```

Comment:
So we don't skip `builtin` directory

Reply:
Please instead change the Makefile and invoke `make lint` here.

Reply:
I'll add to makefil but want to leave this here so we don't get rid of the ci specific parallelization.